ros2_ws/Dockerfile

```diff
@@ -92,6 +108,8 @@ WORKDIR ${HOME}
 RUN wget https://raw.githubusercontent.com/aica-technology/.github/v0.9.0/guidelines/.clang-format
 COPY ./config/config.rviz config.rviz
 COPY --chown=${USER}:${USER} ./config/generate_compile_commands.sh /usr/local/bin/generate_compile_commands
+RUN mkdir -p ${HOME}/.cmake
+COPY --chown=${USER}:${USER} ./config/InstallAicaDescriptions.cmake .cmake/
```

Comment:
Is `~/.cmake` a standard folder which is easy to import? I thought `/usr/share/cmake` was the standard place.